packages/noports_core/lib/src/sshnp/mixins/sshnpd_payload_handler.dart

```diff
@@ -0,0 +1,58 @@
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:at_client/at_client.dart';
+import 'package:meta/meta.dart';
+import 'package:noports_core/sshnp_impl.dart';
+import 'package:noports_core/utils.dart';
+
+mixin DefaultSSHNPDPayloadHandler on SSHNPImpl {
+  @protected
+  late String ephemeralPrivateKey;
+
+  @override
+  FutureOr<bool> handleSshnpdPayload(AtNotification notification) async {
+    if (notification.value?.startsWith('{') ?? false) {
```

Comment:
Deferred the payload to a mixin, since implementation details were leaking into SSHNPImpl, this is the default (AKA non-legacy) payload handler